x509.opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+name: "x509"
+version: "0.6.2"
+maintainer: [
+  "Hannes Mehnert <hannes@mehnert.org>" "David Kaloper <david@numm.org>"
+]
+authors: [
+  "David Kaloper <david@numm.org>" "Hannes Mehnert <hannes@mehnert.org>"
+]
+license: "BSD2"
+tags: "org:mirage"
+homepage: "https://github.com/mirleft/ocaml-x509"
+doc: "https://mirleft.github.io/ocaml-x509/doc"
+bug-reports: "https://github.com/mirleft/ocaml-x509/issues"
+depends: [
+  "ocaml" {>= "4.02.2"}
+  "dune" {build}
```

Comment:
This needs to be `build & >= "1.2"` to match the dune-project version

Reply:
Indeed. Fixed.